api/src/main/kotlin/com/google/devtools/ksp/processing/Resolver.kt

```diff
@@ -37,6 +38,9 @@ interface Resolver {
      */
     fun getAllFiles(): Sequence<KSFile>
 
+    fun getResource(path: String): InputStream?
+    fun getAllResources(): Sequence<String>
```

Comment:
Obviously these would have to be documented before merging.

I also don't know how they would behave on non JVM targets which would have to be tested.